uo_sys_framework/AIDriversModule/cfgXEH/init/ServerPreInit.sqf

```diff
@@ -16,7 +16,7 @@ UO_FW_EXEC_CHECK(CLIENT);
 	if (!(UO_FW_Server_AIDRIVERSMODULE_Allowed)) exitwith {};
 
 	if ((UO_FW_AIDrivers_VehClasses isEqualto "") && {!(UO_FW_GETMVAR(AIDrivers_AllVehs,false))}) exitwith {};
-	["UO_FW_RegisterModuleEvent", ["AI Drivers", "Module for adding AI Drivers to vehicles", "BlackHawk, PiZZADOX and Sacher"]] call CBA_fnc_globalEvent;
+	["UO_FW_RegisterModuleEvent", ["AI Drivers", "Module for adding AI Drivers to vehicles", "BlackHawk, PiZZADOX and Sacher"]] call CBA_fnc_globalEventJIP;
```

Comment:
Why global event JIP? Does this behave normally as a preInit function even as it's referencing JIP?

Reply:
fires for JiP players, executed on server